packages/selinux-policy/lxc_contexts

```diff
@@ -1,3 +1,13 @@
+# Runtimes that use the Go SELinux implementation, such as Docker and
+# the containerd CRI plugin, will apply the 'process' label to the
+# initial process for unprivileged containers, unless the option for
+# automatic labeling is disabled.
 process = "system_u:system_r:container_t:s0"
+
+# The 'file' label should always be applied to the container's root
+# filesystem, whether or not the container is privileged or automatic
+# labeling is disabled.
```

Comment:
Grammatically unclear?

Does it mean?

```
# The 'file' label should always be applied to the container's root
# filesystem, whether or not the container is privileged, or else
# automatic labeling will be disabled.
```

Could be my lack of context...

Reply:
"The file label should always be applied to the container's root, whether or not the container is privileged, unless automatic labeling is disabled."

?

Reply:
Better like this?

```
# The 'file' label should always be applied to the container's root
# filesystem, regardless of whether the container is privileged or
# automatic labeling is disabled.
```

Or this?
```
# The 'file' label should always be applied to the container's root
# filesystem, even if the container is privileged and even when
# automatic labeling is disabled.
```

Reply:
I think A is good.

Reply:
Or this?  :bike: :house:
```
# The 'file' label should always be applied to the container's root
# filesystem, regardless of privileged status or automatic labeling.
```